csvcubed/csvcubed/utils/uri.py

```diff
@@ -107,3 +110,34 @@ def ensure_values_in_lists_looks_like_uris(values: list[str]) -> None:
             raise ValueError(f"'{value}' does not look like a URI.")
 
     _logger.debug("Values %s all look like URIs.", values)
+
+
+def read_from_uri(uri: str, log: logging.RootLogger) -> dict:
```

Comment:
This feels more like a `load_json_from_uri` method. I think it would be better placed with the `file.read_json_from_file` method in a separate module called `json`.

Reply:
Again, I'm not sure why you're accepting a `log` as input param here.